eox_hooks/actions_handler.py

```diff
@@ -51,3 +55,18 @@ def action_lookup(module_name, action):
 
     log.warning(message)
     return custom_action_mock
+
+
+def audit_action_execution(status, sender, action, execution_time=None):
```

Comment:
???

Reply:
This function is now used in action_handler